tests/test_snowflake_common.py

```diff
@@ -256,7 +260,7 @@ def test_get_slice_without_limit_with_offset(result, execute_query, connect, sno
     ds: DataSlice = SnowflakeCommon().get_slice(connect, snowflake_datasource, offset=5)
     assert result.call_count == 3
     assert len(ds.df) == 14
-    assert ds.stats.total_returned_rows == 14
+    assert ds.pagination_info.pagination_info.total_rows == 19
```

Comment:
mhhh, please can you add comments explaining why we're having `19` here ?

Reply:
253521b777b710b4a17adfe1dc517e2b61ed2040